main.js

```diff
@@ -0,0 +1,379 @@
+                if (gameArray[rowCount][columnCount] != null && gameArray[rowCount][columnCount] === gameArray[rowCount+1][columnCount] && gameArray[rowCount][columnCount] === gameArray[rowCount-1][columnCount] && gameArray[rowCount][columnCount] === gameArray[rowCount][columnCount+1] && gameArray[rowCount][columnCount] === gameArray[rowCount][columnCount-1]){
+                    zeusModal();
+                }
+            }
+        }
+    }
+}
+
+function zeusModal() {
+    $(".zeus-modal").removeClass("hidden-modal");
+    $(".zeus-modal-text").text("Praise Zeus mortal for he has granted you one more move");
+    firstPowerUpTrigger = 1;
+    playerSwitch++;
+    $(".zeus-modal").on("click", hideZeus);
+}
+
+function hideZeus() {
+    $(".zeus-modal").addClass("hidden-modal");
+}
+
+function secondPowerUp(someArray){
+    for(var checkRow = someArray.length-1; checkRow >= 2; checkRow--){
+        for(var checkX = 0; checkX <= 4; checkX++){
+            if(someArray[checkRow][checkX] !== null && someArray[checkRow][checkX] === someArray[checkRow-1][checkX+1] && someArray[checkRow-1][checkX+1] === someArray[checkRow-2][checkX+2] && someArray[checkRow-2][checkX+2] === someArray[checkRow-2][checkX] && someArray[checkRow-2][checkX] === someArray[checkRow][checkX+2]){
+                hadesModal();
+            }
+        }
+    }
+}
+
+function hadesModal(){
+    $('.hades-modal-shadow').removeClass('hidden-modal');
+    $('.hades-text').text('You have created an X. Hades will punish you in the underworld for the rest of eternity');
+}
+
+function checkWinCondition() {
+    checkHorizontalWin(gameArray);
+    checkVerticalWin(gameArray);
+    checkDiagonalWin(gameArray);
+    checkDraw();
+}
+
+function checkHorizontalWin(someArray){
+    for(var checkRow = someArray.length-1; checkRow >= 0; checkRow--){
+        for(var checkInnerRow = 0; checkInnerRow < someArray[checkRow].length; checkInnerRow++){
+            if(someArray[checkRow][checkInnerRow] != null && someArray[checkRow][checkInnerRow] === someArray[checkRow][checkInnerRow + 1] && someArray[checkRow][checkInnerRow + 1] === someArray[checkRow][checkInnerRow + 2] && someArray[checkRow][checkInnerRow + 2] === someArray[checkRow][checkInnerRow + 3]){
+                modalWin();
+            }
+        }
+    }
+}
+function checkVerticalWin(someArray){
+    for(var checkRow = someArray.length-1; checkRow >= 4; checkRow--){
+        for(var checkInnerRow = 0; checkInnerRow < someArray[checkRow].length; checkInnerRow++){
+            if(someArray[checkRow][checkInnerRow] != null && someArray[checkRow][checkInnerRow] === someArray[checkRow-1][checkInnerRow] && someArray[checkRow-1][checkInnerRow] === someArray[checkRow-2][checkInnerRow] && someArray[checkRow-2][checkInnerRow] === someArray[checkRow-3][checkInnerRow]){
+                modalWin();
+            }
+        }
+    }
+}
+function checkDiagonalWin(someArray){
+    for(var checkRow = someArray.length-1; checkRow >= 4; checkRow--){
```

Comment:
...similar. Rather than writing an individual function for each, write one loop that takes parameters for it's starting and ending points, and call it as necessary to check for wins. Remember, if you're writing something more than once, there's almost always a better way!